nvidia_rag_mcp/mcp_server.py

```
@@ -320,17 +396,150 @@ async def tool_get_summary(
+        split_options: Optional chunking config, e.g., {'chunk_size':512,'chunk_overlap':150}
+    Returns:
+        dict[str, Any]: Server response JSON or error info.
+    """
+    base_url = _ingestor_base_url()
+    url = f"{base_url}/v1/documents"
+    form_data = aiohttp.FormData()
+    # Add files
+    for path in file_paths or []:
+        try:
+            if os.path.exists(path):
+                with open(path, "rb") as f:
```

Ruff (0.14.8)

Async functions should not open files with blocking methods like `open`

(ASYNC230)